### PR TITLE
lestarch: adding dictionary information to GUI

### PR DESCRIPTION
**Pull request by LeStarch**
| | |
|:---|:---|
|**_Originating Project/Creator_**| With Love from OWLs |
|**_Affected Component_**|  |
|**_Affected Architectures(s)_**|  |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**|  |
|**_Builds Without Errors (y/n)_**|  |
|**_Unit Tests Pass (y/n)_**|  |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

Adds dictionary information to the GUI.

### CODE REVIEW

File: src/fprime_gds/flask/static/addons/dictionary/addon.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+});
+Vue.component("dictionary-version", {
+    template: version_template,
+    data() {
+        return {
+            "framework_version": _dictionaries.framework_version,
+            "project_version": _dictionaries.project_version
+        }
+    }
+})
```
Comment (Joshua-Anderson) [inline src/fprime_gds/flask/static/addons/dictionary/addon.js]: Nitpick: This file should probably have a newline at the end.

**Review by Joshua-Anderson — DISMISSED** [review]
Looks awesome! I had fun quickly testing this locally.

I noticed that we didn't have any dictionary comments on the ref app - it looks like all the dictionary comments were removed with the transition to fpp. Does FPP support dictionary comments?

**Comment by LeStarch** [comment]
@Joshua-Anderson I just fixed the nitpick.  Once more over and we'll merge?  I think FPP supports dictionary comments, but I am not sure.

**Review by Joshua-Anderson — APPROVED** [review]
:shipit:

**Pull request merged**